VirtoCommerce.MarketingModule.Data/Model/PublishingGroupContentItemEntity.cs

```diff
@@ -1,22 +1,58 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Runtime.Serialization;
-using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
+using System;
+using VirtoCommerce.Domain.Marketing.Model;
 using VirtoCommerce.Platform.Core.Common;
 
 namespace VirtoCommerce.MarketingModule.Data.Model
 {
-	public class PublishingGroupContentItemEntity : AuditableEntity
-	{
-		#region Navigation Properties
-		public string DynamicContentPublishingGroupId { get; set; }
-		public virtual DynamicContentPublishingGroupEntity PublishingGroup { get; set; }
+    public class PublishingGroupContentItemEntity : AuditableEntity
+    {
+        #region Navigation Properties
+        public string DynamicContentPublishingGroupId { get; set; }
+        public virtual DynamicContentPublishingGroupEntity PublishingGroup { get; set; }
 
         public string DynamicContentItemId { get; set; }
-		public virtual DynamicContentItemEntity ContentItem { get; set; }				
-		#endregion
-	}
+        public virtual DynamicContentItemEntity ContentItem { get; set; }				
+        #endregion
+
+        public int Priority { get; set; }
+
+        public DynamicContentItem ToModel(DynamicContentItem contentItem)
```

Comment:
I'm not sure what you need a pair FromModel and ToModel for this 'bridge' type

Reply:
Definitely we need. If we working with content items in group context. For example for prioritization in the groups.